package/yast2-python-bindings.changes

```diff
@@ -1,3 +1,13 @@
+-------------------------------------------------------------------
+Thu May  5 11:27:01 UTC 2022 - Noel Power <nopower@suse.com>
+
+- Misc fixes to examples; (bsc#1199343).
+- Fix python2 utf8 handling (python2 code wasn't displaying unicode
+  strings); (bsc#1199348).
+- Change conversion of Py_None to YCP_Null instead of YCP_Void;
```

Comment:
The change in src/ytypes.i does the opposite of what you describe here in the changes.
Your comment says it changes "Py_None to YCP_Null instead of YCP_Void", but src/ytypes.i changes Py_None from YCPNull to YCPVoid.
Changing it to YCPVoid makes sense (now functions return void instead of NULL), so I think you just need to fix your comment in the changelog.